index.html

```diff
@@ -3,18 +3,46 @@
     <head>
         <meta charset="utf-8">
         <meta name="google-site-verification" content="XYxEjxEMJ-lflwv40c2SiK5UsgTeMnMN0xrWCLww1es">
-        <link href='//fonts.googleapis.com/css?family=Open+Sans:300' rel='stylesheet' type='text/css'>
         <link href="css/index.css" rel="stylesheet" type="text/css">
         <title>Copilot</title>
     </head>
     <body>
+        <div id="info">
+            <div id="info-wrapper">
+                <center>
+                    <h1>About Copilot</h1>
+                    <br />
+                </center>
+                    <p>
+                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bacon ipsum dolor amet kielbasa chuck short loin, tenderloin sirloin beef ribs biltong jerky ground round leberkas pork chop. Hamburger ball tip turducken, chuck cupim bresaola beef pork loin prosciutto jerky pig boudin. Filet mignon pork chop frankfurter, pork loin kevin tri-tip porchetta spare ribs turducken andouille. Corned beef ground round andouille, pancetta boudin meatloaf ribeye meatball landjaeger cow filet mignon strip steak porchetta drumstick kevin. Swine ham landjaeger, venison fatback andouille bacon alcatra tail doner pork belly rump beef hamburger. Andouille shank landjaeger doner porchetta rump pancetta.
+                    </p>
+                    <br />
+                    <p>
+                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bresaola shoulder sausage, alcatra swine ground round prosciutto strip steak rump. Chuck meatball ham pancetta pork belly. Turducken brisket swine landjaeger pork short loin. Capicola jerky rump andouille ham. Pastrami cupim turducken pork jerky t-bone, turkey pork chop capicola kielbasa beef brisket shoulder ham hock cow. Pork chop landjaeger shankle pancetta, kevin flank prosciutto porchetta leberkas tenderloin corned beef jerky short ribs.
+                    </p>
+                    <br />
+                    <p>
+                        Love,
+                        <br />
+                        Team Copilot
+                    </p>
+                <center>
+                    <br /><br />
+                    <button id="back">Home&nbsp;<i class="icon ion-home" style="color: #191919;"></i></button>
+                </center>
+            </div>
+        </div>
+
         <section class="centered">
             <h1>Copilot</h1>
-            <h3>We're here to help. <a href="#">Learn more ›</a></h3>
+            <h3>We're here to help. <a id="more" href="#">Learn more <span>&rsaquo;</span></a></h3>
```

Comment:
Yes, it's redundant. From a user's standpoint it'll make no difference. I personally like using ```&rsaquo;``` because I'm always sure what the character is when writing code in my editor.